resources/polygons.geojson

```diff
@@ -3,8 +3,10 @@
 "name": "polygons",
 "crs": { "type": "name", "properties": { "name": "urn:ogc:def:crs:OGC:1.3:CRS84" } },
 "features": [
-{ "type": "Feature", "properties": { "id": null, "class": "FOREST", "color": "Black" }, "geometry": { "type": "MultiPolygon", "coordinates": [ [ [ [ 0.926738329283575, 43.872984835004949 ], [ 0.94793262988151, 43.922435753652707 ], [ 1.14067868090431, 43.891387901356886 ], [ 1.144922923707132, 43.809266649405295 ], [ 0.952906187174365, 43.826841630979345 ], [ 0.926738329283575, 43.872984835004949 ] ] ] ] } },
-{ "type": "Feature", "properties": { "id": null, "class": "WATER", "color": "Blue" }, "geometry": { "type": "MultiPolygon", "coordinates": [ [ [ [ 1.007413493443309, 43.716326190738407 ], [ 1.000101365965903, 43.732759423626817 ], [ 1.05303497919675, 43.743794288627157 ], [ 1.093391035206889, 43.725153673450073 ], [ 1.062854785638347, 43.701638596581482 ], [ 0.981759516082789, 43.712195696118414 ], [ 1.007413493443309, 43.716326190738407 ] ] ] ] } },
-{ "type": "Feature", "properties": { "id": null, "class": "FARMLAND", "color": "Green" }, "geometry": { "type": "MultiPolygon", "coordinates": [ [ [ [ 0.804435668608206, 43.774709103468517 ], [ 0.844193029411034, 43.775466280235008 ], [ 0.865770743441819, 43.739989613092405 ], [ 0.831338832243599, 43.721246898232536 ], [ 0.76424372591599, 43.728847383084364 ], [ 0.765078925893384, 43.751864880600344 ], [ 0.804435668608206, 43.774709103468517 ] ] ] ] } }
+{ "type": "Feature", "properties": { "id": null, "class": "WATER", "color": "blue" }, "geometry": { "type": "Polygon", "coordinates": [ [ [ 1.428090901495726, 43.602284563088126 ], [ 1.428089026885723, 43.602356545315459 ], [ 1.429040215614357, 43.603881791155978 ], [ 1.434488292290918, 43.60395654390355 ], [ 1.440126601590054, 43.600505239754945 ], [ 1.439821106746437, 43.593156244512485 ], [ 1.436059506933373, 43.593032827932589 ], [ 1.438266986990658, 43.59961574857978 ], [ 1.434837726552646, 43.601945105233732 ], [ 1.428090901495726, 43.602284563088126 ] ] ] } },
```

Comment:
I see that you changed an existing `polygons.geojson` ; would it be possible to revert this change, and give a different name to this file, to preserve the original one?
You can also create a subfolder in `resources/nbertaud` for instance to store all your files